### PR TITLE
Quizes

### PR DESCRIPTION
**Pull request by kaleem99**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [ ] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by ry-oc — APPROVED** [review]
looks good :)

File: content/learning-how-to-teach-coders/quizzes/_index.md [hunk 1]
```diff
@@ -1,5 +1,45 @@
 ---
-_db_id: 649
```
Comment (sheenarbw) [inline content/learning-how-to-teach-coders/quizzes/_index.md:2]: why did you delete this line? 

If _db_id is there: dont do anything to it

File: content/learning-how-to-teach-coders/quizzes/_index.md [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
 content_type: topic
 title: Quizzes
----
\ No newline at end of file
+---
+
+## What The Sessions Should Be About
```
Comment (sheenarbw) [inline content/learning-how-to-teach-coders/quizzes/_index.md]: The following heading is much easier on the eye. It's generally good to just use sentence case unless there is a word that needs extra emphasis

```
## What the session should be about
```

File: content/learning-how-to-teach-coders/quizzes/_index.md [hunk 3]
```diff
@@ -1,5 +1,45 @@
...
+
+<!-- This type of session involves the leader and follower working through some kind of Q&A type quiz. The topic could be very basic or something more advanced. -->
+
+The session should be about a junior staff members or jtl giving the recruits some questions, quizzes on various topics to work through and see if recruits have a strong and good understanding on the basics foundational skills.
```
Comment (sheenarbw) [inline content/learning-how-to-teach-coders/quizzes/_index.md]: What if there is another code school making use of our open source content? Will they understand this session?

Should we not just say that there is a leader and a follower and the leader needs to give the follower some questions etc? Why superglue this to the Umuzi context?

File: content/learning-how-to-teach-coders/quizzes/_index.md [hunk 4]
```diff
@@ -1,5 +1,45 @@
...
+If recruits are struggling with some concepts and cannot think of solutions to breaking problems down take them back to the basic concepts and see if they understand everything, if they have a good understanding of the basics then the main focus for them should be to practice and learn problem solving techniques.
+
+## Resources
+- [Python Quizzes](https://realpython.com/quizzes/)
+- [Data science questions](https://github.com/alexeygrigorev/data-science-interviews)
+- [CodeQuizzes](https://www.codequizzes.com/)
+- [JavaScript Quiz](https://www.w3schools.com/js/js_quiz.asp)
+- [Java Quiz](https://www.w3schools.com/java/java_quiz.asp)  
+
+## Practice Problem Solving
```
Comment (sheenarbw) [inline content/learning-how-to-teach-coders/quizzes/_index.md]: Are these useful here? Should they be somewhere else? Maybe these problem solving resources would be more appropriate in the  'debugging"  topic (content/learning-how-to-teach-coders/debugging/_index.md).  

These are good resources but I'm not sure that they are good quiz resources. Our syllabus is like code, it should be DRY and Cohesive.

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Nice work! but it does have some rough edges.

File: content/learning-how-to-teach-coders/quizzes/_index.md (hunk 1 above)
Comment (ry-oc) [reply content/learning-how-to-teach-coders/quizzes/_index.md:2]: @kaleem99 thats my bad, i didnt realise the _db_id was autogenerated as I thought this was a completely new page, apologies

**Review by sheenarbw — APPROVED** [review]
